src/api/upload-file.ts

```diff
@@ -0,0 +1,44 @@
+import {
+    Request,
+    Response,
+} from 'express';
+
+import {
+    uploadFileConfirmationCall,
+    uploadFileConfirmationSubmit,
+} from '../forms/upload-google';
+import {
+    AppCallResponse,
+} from '../types';
+import {
+    CallResponseHandler,
+    newFormCallResponse,
+    newOKCallResponse,
+} from '../utils/call-responses';
+import {
+    showMessageToMattermost,
+} from '../utils/utils';
+
+export const uploadFileToDriveCall: CallResponseHandler = async (req: Request, res: Response) => {
+    let callResponse: AppCallResponse;
+
+    try {
+        const form = await uploadFileConfirmationCall(req.body);
```

Comment:
I think this is related to this [comment](https://github.com/mattermost/mattermost-app-google-drive/pull/2#discussion_r1053868933)

In case it is, it was resolved in commit [295cf1c](https://github.com/mattermost/mattermost-app-google-drive/pull/2/commits/295cf1c212d7931cfd516e017dcd2e7e23d1929a)